diff --git a/node_modules/@nozbe/watermelondb/native/android/build.gradle b/node_modules/@nozbe/watermelondb/native/android/build.gradle
index 90ddf7a..62c3fe9 100644
--- a/node_modules/@nozbe/watermelondb/native/android/build.gradle
+++ b/node_modules/@nozbe/watermelondb/native/android/build.gradle
@@ -13,7 +13,7 @@ buildscript {
     }

     dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${getExtOrDefault('kotlinVersion')}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50")
     }
 }

@@ -35,7 +35,7 @@ android {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getExtOrDefault('kotlinVersion')}"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.50"
 }

 repositories {
